lago/ssh.py

```diff
@@ -354,7 +359,7 @@ def get_ssh_client(
                 )
 
             ssh_tries -= 1
-            time.sleep(1)
+            time.sleep(5)
```

Comment:
This change is totally alien to the pr too, pleas split into another pr (or explain the relation to the custom internal repo)